optimum/exporters/neuron/config.py

```diff
@@ -53,3 +56,39 @@ class TextAndVisionNeuronConfig(NeuronConfig):
     """
 
     DUMMY_INPUT_GENERATOR_CLASSES = (DummyTextInputGenerator, DummyVisionInputGenerator, DummyBboxInputGenerator)
+
+
+class DecoderNeuronConfig(ExportConfig):
+    """
+    Handles decoder architectures.
+    """
+
+    NEURONX_ARGS = {"batch_size": 1, "n_positions": 128, "tp_degree": 2, "amp": "f32"}
+    NEURONX_MODULE = None
+    NEURONX_CLASS = None
+
+    def __init__(self, task, **kwargs):
```

Comment:
```suggestion
    def __init__(self, task: str):
```

Why allow for keyword arguments here?

Reply:
I think this is a left-over. I will remove it